Makefile

```diff
@@ -80,9 +82,6 @@ fmt: ## Run go fmt against code.
 lint: golangci-lint ## Run golangci-lint against code.
 	GOOS=$(TARGET_OS) CGO=1 $(GOLANGCI_LINT) run ./...
 
-.PHONY: check
-check: manifests generate add-license lint test
```

Comment:
Is there a reason why the `check` has been removed? As we have a `check` directive in all other projects I would keep it.

Reply:
Added back this check. Indeed, required in makefile.